coalib/bearlib/abstractions/Linter.py

```diff
@@ -447,10 +447,17 @@ def run(self, filename, file, **kwargs):
                     FunctionMetadata.filter_parameters(
                         self._get_create_arguments_metadata(), kwargs))
 
-                arguments = (self.get_executable(),) + tuple(
-                    self.create_arguments(
-                        filename, file, config_file,
-                        **create_arguments_kwargs))
+                args = self.create_arguments(filename, file, config_file,
+                                             **create_arguments_kwargs)
+
+                try:
+                    args = tuple(args)
+                except TypeError:
+                    self.err("The given arguments "
+                             "`{}` are not iterable.".format(args))
```

Comment:
rather use `{0!r}` to let python use repr on args instead of putting single quotes around it manually